tiledb/bioimg/converters/scale.py

```diff
@@ -96,18 +81,15 @@ def apply(self, base: tiledb.Array, output: tiledb.Array, level: int) -> None:
             )
         else:
             with Pool(multiprocessing.cpu_count()) as pool:
```

Comment:
1. No need for `multiprocessing.cpu_count()`, that's the default
2. The [`concurrent.futures`](https://docs.python.org/3/library/concurrent.futures.html#processpoolexecutor-example) module is slightly preferred to `multiprocessing` these days. Since you don't need the results of the parallelized function, you can [submit](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.Executor.submit) each callable to be scheduled and finally [wait](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.wait) for all of them to finish.

Reply:
Another idea: add an optional `max_workers` parameter (default=`multiprocessing.cpu_count()`) in `pyramid_kwargs` just like the one in `ImageConverter.to_tiledb` to allow more flexibility. May also allow `max_workers=0` to mean sequential scaling, one tile at a time.